server/tests/test_mpesa_routes.py

```
@@ -0,0 +1,314 @@
+
+        response = client.post(
+            "/api/stk-push",
+            data=json.dumps(data),
+            content_type="application/json",
+        )
+
+        assert response.status_code == 400
+        json_data = response.get_json()
+        assert "error" in json_data
+
+    def test_initiate_stk_push_invalid_phone(self, client, session):
```

Ruff (0.13.1)

Unused method argument: `session`

(ARG002)